rules-reviewed/jboss-eap/websphere/websphere.windup.xml

```diff
@@ -0,0 +1,701 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/v1/xml" id="websphere_rules_general">
+    <rules>
+        <rule id="websphere_rules_general-7Apvvd">
```

Comment:
Is the ruleset id supposed to be the same as the filename? I don't know if we established that.